src/fusets/peakvalley.py

```diff
@@ -0,0 +1,168 @@
+from datetime import datetime
+from typing import Iterable, Sequence, Tuple
+
+import numpy as np
+import xarray
+from scipy.signal import find_peaks
+from xarray import DataArray
+
+from fusets._xarray_utils import _extract_dates, _time_dimension
+
+
+def peakvalley(
+    array: DataArray,
+    drop_thr: float = 0.15,
+    rec_r: float = 1.0,
+    slope_thr: float = -0.007,
+) -> DataArray:
+
```

Comment:
This is the main entry point for peak-valley, so some (initial) API docs here would be good.